examples/metal-nimf-2-ibm2-connection/main.tf

```diff
@@ -71,3 +72,7 @@ resource "ibm_dl_gateway_action" "test_dl_gateway_action" {
   metered         = var.ibm_gateway_metered
   resource_group  = data.ibm_resource_group.rg.id
 }
+
+data "equinix_metal_connection" "NIMF-test" {
+  connection_id     = equinix_metal_connection.metal-connection.id
```

Comment:
Add depends_on = [time_sleep.wait_dl_connection] line in Metal Connection data source block